src/PointwiseFunctions/ScalarTensor/RampUpFunction.cpp

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/ScalarTensor/RampUpFunction.hpp"
+
+#include "Utilities/ConstantExpressions.hpp"
+
+double ScalarTensor::fifth_derivative_ramp_function(
+    const double time, const double start_time, const double ramp_time) {
+  double ramp_factor = 1.0;
+
+  if (time < start_time) {
+    // Before ramping on
+    ramp_factor = 0.0;
+  } else if (time < ramp_time + start_time) {
+    // Ramping on with ramp function given in Eq. B4 of 1911.02588
+    const double t = (time - start_time) / ramp_time;
+    ramp_factor =
+        pow<5>(t) * (126. + t * (-420. + t * (540. + t * (-315. + 70. * t))));
```

Comment:
We have a function `integer_pow` in `src/Utilities/Math.hpp` which I think will be more efficient than the `pow<5>(t)` which I think just expands to `t * t * t * t * t`